src/api/getMovieFromServer.js

```diff
@@ -0,0 +1,14 @@
+const BASE_URL = `https://www.omdbapi.com/?apikey=5a5359e&t=`;
```

Comment:
better to call this file just `api.js`
It may potentially have some other requests than get movies